### PR TITLE
Document /api/[de]authorize-user

### PR DESCRIPTION
**Pull request by heyitsmeuralex**
Ping @TheInitializer; you were the one to realise this.

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -813,6 +813,44 @@ GET /api/username-available/patrick
 <- }
 ```
 
+<a name='authorize-user'></a>
+### Authorize a user [POST /api/authorize-user]
++ requires admin session
++ userID (ID)
+
+On success, returns `{ authorized: true }`. Note that this endpoint will error (`AUTHORIZATION_ERROR`) if the server does not [require authorization](#authorization).
```
Comment (towerofnix) [inline docs/api.md]: It doesn't return `{authorized: true}`, it returns `{}`.

File: docs/api.md [hunk 2]
```diff
@@ -813,6 +813,44 @@ GET /api/username-available/patrick
...
+<-   "authorized": true
+<- }
+```
+
+<a name='authorize-user'></a>
+### Deauthorize a user [POST /api/deauthorize-user]
++ requires admin session
++ userID (ID) - can't be you
+
+On success, returns `{ authorized: false }`. Note that this endpoint will error (`AUTHORIZATION_ERROR`) if the server does not [require authorization](#authorization) or if you attempt to deauthorize yourself.
```
Comment (towerofnix) [inline docs/api.md]: Same deal here - it doesn't return false, it returns `{}`.

File: docs/api.md [hunk 3]
```diff
@@ -813,6 +813,44 @@ GET /api/username-available/patrick
...
+```js
+POST /api/deauthorize-user
+
+-> {
+->   "userID": "123456"
+-> }
+
+<- {
+<-   "authorized": false
+<- }
```
Comment (towerofnix) [inline docs/api.md]: Make sure you fix the example blocks, too!

**Comment by heyitsmeuralex** [comment]
Oops, I read the source wrong :package: - should be fixed now.

**Pull request merged**